extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -348,8 +368,17 @@ export async function getSelectedSubscriptions(appContext: AppContext, account?:
  * @param host Use this to override the host. The default host is https://management.azure.com
  * @param requestHeaders Provide additional request headers
  */
-export async function makeHttpRequest(account: AzureAccount, subscription: azureResource.AzureResourceSubscription, path: string, requestType: HttpRequestMethod, requestBody?: any, ignoreErrors: boolean = false, host: string = 'https://management.azure.com', requestHeaders: { [key: string]: string } = {}): Promise<AzureRestResponse> {
-	const result: AzureRestResponse = { response: {}, errors: [] };
+export async function makeHttpRequest<B>(
+	account: AzureAccount,
+	subscription: azureResource.AzureResourceSubscription,
+	path: string,
+	requestType: HttpRequestMethod,
+	requestBody?: any,
+	ignoreErrors: boolean = false,
+	host: string = 'https://management.azure.com',
```

Comment:
can this become a common constant for the default management url

Reply:
You're saying like put it in azurecore.d.ts?

Reply:
or create a common referenced constant internally to simplify reuse

Reply:
@cheenamalhotra @cssuh We shouldn't even have this as a default, right? It won't work for other clouds. Can you make an issue for following up on this (if one doesn't exist)?